chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
Do I need modify the docs?
Or simply add a comment as follows.
```       
        # use -1 instead of None internally,
        # since some serializers do not support None.
        self._previous_epoch_detail = -1.
```